service/storage_service.cc

```diff
@@ -385,6 +388,12 @@ future<> storage_service::topology_state_load(cdc::generation_service& cdc_gen_s
                 return read_new_t::no;
             }
             switch (*state) {
+                case topology::transition_state::tablet_allow_write_both_read_old:
```

Comment:
Don't all of these need `[[fallthrough]]`?

Reply:
Right. The compilation didn't fail though.